test/interpolation1D.jl

```diff
@@ -17,7 +17,6 @@ function test_single_column(
     trial_data = nothing
     @testset "1D linear interpolation on single column with $FT" begin
         toler = FT(0.003)
```

Comment:
Can we use `tol` that's based on `eps(FT)`

Reply:
Tolerance here more a function of grid spacing rather than  `FT`.